src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingExerciseResource.java

```diff
@@ -308,8 +308,8 @@ public ProgrammingExerciseResource(ProgrammingExerciseRepository programmingExer
         }
 
         // Check if package name is set
-        if (programmingExercise.getProgrammingLanguage() == ProgrammingLanguage.JAVA) {
-            // only Java needs a valid package name at the moment
+        if (programmingExercise.getProgrammingLanguage() == ProgrammingLanguage.JAVA || programmingExercise.getProgrammingLanguage() == ProgrammingLanguage.KOTLIN) {
+            // only Java and Kotlin need a valid package name at the moment
```

Comment:
Just as a small note: I believe Kotlin handles Identifiers differently to Java and is way more permissive. I don't know if this applies to package names, but it is quite likely that different rules apply (different character groups, whitespace, and keywords). However, I don't think that the package names are that different in practice, so we should be fine here for now (we should keep that in mind though).